fluentui_drawer/src/main/java/com/microsoft/fluentui/bottomsheet/BottomSheetAdapter.kt

```diff
@@ -67,6 +68,7 @@ class BottomSheetAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder> {
 
     private inner class BottomSheetItemViewHolder : RecyclerView.ViewHolder {
         private val listItemView: ListItemView
+        private var image: ImageView ?= null
```

Comment:
Is this reference really needed?

Reply:
Moving this to the scope of function where it is needed.